dataland-frontend/src/components/resources/frameworkDataSearch/sfdr/SfdrDataModel.ts

```diff
@@ -88,7 +87,8 @@ export const sfdrDataModel = [
           {
             name: "scope1GhgEmissionsInTonnes",
             label: "Scope 1 GHG emissions",
-            description: "What is the amount of the company's Scope 1 emissions?",
+            description:
```

Comment:
<img width="405" alt="SFDR2" src="https://github.com/d-fine/Dataland/assets/104447806/7c526366-125c-4745-aaa1-4b190d438acf">
General question: Should total revenue and enterprise value be listed under Greenhouse Gas Emissions?

Reply:
I asked Julia about this and she said that it would indeed make sense to leave them here under GHG - at least in SFDR, since these fields are apparently used to compute other GHG-related things like the intensity and some carbon-related indicators